### PR TITLE
Move dropbox vendors

### PR DESCRIPTION
**Pull request by Mottie**
Moved dropbox vendor files (dropbox sdk & zipjs) into the main vendor folder and added an update method.

@matheusfaustino I'm not sure how to test dropbox locally after moving the folders.

### CODE REVIEW

**Comment by matheusfaustino** [comment]
Hi @Mottie , nice, thank you. I will test for you ^^ 
I'm going to record the process to show you how we can test it locally.

**Comment by Mottie** [comment]
@matheusfaustino were you able to test this PR? We'd like to get a new release out very soon.

**Comment by matheusfaustino** [comment]
@Mottie tonight ^^

File: manage.html [hunk 1]
```diff
@@ -168,8 +168,8 @@ <h2 class="style-name">
   <script src="js/sections-util.js"></script>
   <script src="js/storage-util.js"></script>
 
-  <script src="sync/vendor/dropbox/dropbox-sdk.js" async></script>
-  <script src="sync/vendor/zipjs/zip.js" defer></script>
+  <script src="vendor/dropbox/dropbox-sdk.js" async></script>
+  <script src="vendor/zipjs/zip.js" defer></script>
```
[the remark this answers is not in the document]
Comment (Mottie) [reply manage.html]: Oops! Fixed.

**Comment by matheusfaustino** [comment]
After the merge, if you want to I can test it again.

**Comment by Mottie** [comment]
I think we're okay with fetch - https://caniuse.com/#search=fetch

**Comment by Mottie** [comment]
Ok, try it now. And thanks again for testing this!

I'd appreciate some detailed instruction on how to test this so I don't have to bug you... maybe you could make a wiki page? 😸

**Comment by matheusfaustino** [comment]
That solved the warning. 
Actually, I'm working on a video right now to show you how to test it. I'll post it here, and if it good enough, I can make a wiki page explaining better.

**Comment by Mottie** [comment]
Awesome! Thanks! 

So can I merge this PR?

**Comment by matheusfaustino** [comment]
Yeap :)

**Comment by matheusfaustino** [comment]
hey @Mottie , what do you think about this video? It shows something about the dropbox key and how to change it and how to get the URL for Dropbox. I did it with firefox, but it's quite similar with chrome. 

https://send.firefox.com/download/3ed5dc179b/#47zQvCRTk26ZpnBIWUER5w

**Comment by Mottie** [comment]
Hey @matheusfaustino!

We just released v1.5.1 and I'm already getting reports of dropbox not working. 

The development mode popup would show instead of connecting for *new* Firefox/Chrome installs. So that was fixed in https://github.com/openstyles/stylus/commit/9fe721945d0770b020cf09ef5cf6bb6197dc87eb; but the Dropbox auth popup never opens, at least not in Firefox - I didn't test a new Chrome install.

I noticed on another computer that the Dropbox auth popup opens in a FF window (it's an OS window for Chrome. Any ideas?

**Comment by matheusfaustino** [comment]
I tested in a fresh firefox installation and the development popup is showing too.

**Comment by DanaMW** [comment]
Dropbox works side loaded now good, couple of bitches, sorry, it only allows one copy of save, would like a choice. when you import  some note should be made that it overwrites without asking, you click import you get it even if you didn't expect it. other than that nice work.

**Comment by matheusfaustino** [comment]
Chrome login is working too @Mottie after your advice 
selection_001

**Comment by Mottie** [comment]
Ok, let's open a new issue about the slow to never loading that's being seen.

@DanaMW We've talked about adding more options for Dropbox... we need to open an issue to keep track of it.

**Pull request merged**